homeassistant/components/deconz/gateway.py

```diff
@@ -164,15 +164,14 @@ async def options_updated(self):
             else:
                 deconz_ids += [group.deconz_id for group in groups]
 
-        if deconz_ids:
```

Comment:
@Kane610 can you confirm that this is ok? When an entry is removed from the entity registry, Home Assistant will already remove the corresponding entity, so this signal isn't necessary.

Reply:
Looks ok, the only thing would be if something were not to have a unique id I suppose. But to my knowledge everything does have that